tests/monitors/expvar/expvar_test.py

```diff
@@ -62,6 +62,7 @@ def test_expvar_custom_metric():
               port: 8080
               metrics:
               - JSONPath: queues.count
+                pathSeparator: '.'
```

Comment:
If you have to add this to make this integration test work, then it means it isn't backwards compatible and you will break existing deployments.

Reply:
The character '.' is now the default as before, but somehow the path separator is not set if you don't explicitly configure it in your metrics section. If you don't have a metrics section all together in your config then the default path separator is set!

Reply:
@keitwb I implemented a temp fix for this by setting the default path separator value manually.